src/frontend/packages/core/src/components/design-system/Modal/index.tsx

```diff
@@ -58,8 +58,7 @@ export const MagnifyModal = ({
               props.validateButtonCallback?.();
             }}
           >
-            props.validateButtonLabel ?? intl.formatMessage(commonMessages.yes)
-            {intl.formatMessage(commonMessages.cancel)}
+            {props.validateButtonLabel ?? intl.formatMessage(commonMessages.yes)}
```

Comment:
It means we are missing a test? add a test to secure it?

Reply:
+1